### PR TITLE
perform data shredding

### PR DESCRIPTION
**Pull request by abhiabhi2306**
This will perform shredding on C:/ drive on the deleted data and overwrite that making it impossible to recover the deleted data, this is helpful in scenarios where you need to make deleted data unrecoverable.

### CODE REVIEW

File: execution/c9127250-9ec7-4d6c-b673-028a6191cbf9.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+name: Shred deleted data
+description: |
+  This will perform shredding on C:/ drive on the deleted data and overwrite that making it impossible to recover the deleted data, this is helpful in scenarios where you need to make deleted data unrecoverable.
+technique:
+  id: T1082
+  name: Command and Scripting Interpreter
+platforms:
+  windows:
+    cmd:
+      command: ciper /w:C:\
```
Comment (w0rk3r) [inline execution/c9127250-9ec7-4d6c-b673-028a6191cbf9.yml:16]: @abhiabhi2306 I think you meant `cipher` instead of `ciper`

**Review by w0rk3r — COMMENTED** [review]
@abhiabhi2306 I think you meant `cipher` instead of `ciper`

**Comment by abhiabhi2306** [comment]
Oops, typo! 


Fixed, thank you.

**Review by privateducky — CHANGES_REQUESTED** [review]
let's add a tag of "destructive" to this, which will disable the TTP by default. This is a simple protective measure we use to ensure a user doesn't accidentally bork themselves.

**Comment by abhiabhi2306** [comment]
> let's add a tag of "destructive" to this, which will disable the TTP by default. This is a simple protective measure we use to ensure a user doesn't accidentally bork themselves.

Yes, thats right, added.

Thank you

File: execution/c9127250-9ec7-4d6c-b673-028a6191cbf9.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+id: c9127250-9ec7-4d6c-b673-028a6191cbf9
+metadata:
+  version: 1
+  authors: 
+    - Abhishek S (abhiabhi2306)
+  tags: 
+    - Destructive
```
Comment (khyberspache) [inline execution/c9127250-9ec7-4d6c-b673-028a6191cbf9.yml:7]: lowercase for consistency

File: execution/c9127250-9ec7-4d6c-b673-028a6191cbf9.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+name: Shred deleted data
+description: |
+  This will perform shredding on C:/ drive on the deleted data and overwrite that making it impossible to recover the deleted data, this is helpful in scenarios where you need to make deleted data unrecoverable.
+technique:
```
Comment (khyberspache) [inline execution/c9127250-9ec7-4d6c-b673-028a6191cbf9.yml:11]: Missing `tactic`

Lets change this TTP to a technique in the impact tactic

**Review by khyberspache — CHANGES_REQUESTED** [review]

**Comment by abhiabhi2306** [comment]
added tactic, and changed path.

thank you :)

File: impact/c9127250-9ec7-4d6c-b673-028a6191cbf9.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+id: c9127250-9ec7-4d6c-b673-028a6191cbf9
+metadata:
+  version: 1
+  authors: 
+    - Abhishek S (abhiabhi2306)
+  tags: 
+    - Destructive
```
Comment (khyberspache) [inline impact/c9127250-9ec7-4d6c-b673-028a6191cbf9.yml:7]: lowercase for consistency

File: impact/c9127250-9ec7-4d6c-b673-028a6191cbf9.yml [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+name: Shred deleted data
+description: |
+  This will perform shredding on C:/ drive on the deleted data and overwrite that making it impossible to recover the deleted data, this is helpful in scenarios where you need to make deleted data unrecoverable.
+tactic: impact
+technique:
+  id: T1082
+  name: Command and Scripting Interpreter
```
Comment (khyberspache) [inline impact/c9127250-9ec7-4d6c-b673-028a6191cbf9.yml:14]: lets make this:
```
id: T1485
name: Data Destruction
```

**Review by khyberspache — CHANGES_REQUESTED** [review]

**Comment by abhiabhi2306** [comment]
Done, thank you :)

**Pull request merged**